spec/support/mock_server.rb

```diff
@@ -2,7 +2,7 @@
 require 'byebug'
 
 set :port, 3000
-set :bind, '0.0.0.0'
+set :bind, 'localhost'
```

Comment:
This is not good. It ends up listening on 127.0.0.1 for me, which is the default (when not setting bind), and it does not lets me access the server in mt vagrant setup.

What is the issue in leaving it `0.0.0.0` exactly?